client/js/views/templates/Footer.js

```diff
@@ -1,11 +1,11 @@
 module.exports = function() { 
     return `<footer>
-    <div> future days farm </div>
+    <div>future days farm</div>
     <div> 
-        2123 Tiny Road <br> Town Name, Michigan 33344 <br> <br>
-        <a href = "mailto:Info@FutureDaysFarm.com"> Info@FutureDaysFarm.com </a> <br>
+        2123 Tiny Road<br>Town Name, Michigan 33344 <br><br>
+        <a href="mailto:Info@FutureDaysFarm.com"> Info@FutureDaysFarm.com</a><br>
```

Comment:
Remove the space before `Info`.

Also, I believe its best practice to employ a self closing `<br />`.